thorlcr/mfilemanager/thmfilemanager.cpp

```diff
@@ -504,39 +502,49 @@ class CFileManager : public CSimpleInterface, implements IThorFileManager
             if (nonLocalIndex)
                 ++total;
             StringBuffer dir;
+            bool dirPerPart = false;
             if (temporary && !job.queryUseCheckpoints()) 
                 dir.append(queryTempDir(false));
             else
             {
                 // NB: always >= 1 groupNames
                 StringBuffer curDir;
+                bool curDirPerPart = false;
```

Comment:
minor: Clearer if this is within the loop below (see other changes)

Reply:
agree, have changed and separated out the tmp dir var. for clarity, please see sep. (latest) commit.